cmplxs/cmplxs_test.go

```diff
@@ -725,6 +725,62 @@ func TestMul(t *testing.T) {
 	}
 }
 
+func TestMulConj(t *testing.T) {
+	s1 := []complex128{1 + 1i, 2 + 2i, 3 + 3i}
+	s2 := []complex128{1 + 1i, 2 + 2i, 3 + 3i}
+	ans := []complex128{2, 8, 18}
+	MulConj(s1, s2)
+	if !EqualApprox(s1, ans, EqTolerance) {
+		t.Errorf("MulConj doesn't give correct answer. Expected %v, Found %v", ans, s1)
+	}
+	s1short := []complex128{1}
+	if !Panics(func() { MulConj(s1short, s2) }) {
+		t.Errorf("Did not panic with unequal lengths")
+	}
+	s2short := []complex128{1}
+	if !Panics(func() { MulConj(s1, s2short) }) {
+		t.Errorf("Did not panic with unequal lengths")
+	}
+}
+
+func TestMulConjTo(t *testing.T) {
+	s1 := []complex128{1 + 1i, 2 + 2i, 3 + 3i}
+	s1orig := []complex128{1 + 1i, 2 + 2i, 3 + 3i}
+	s2 := []complex128{1 + 1i, 2 + 2i, 3 + 3i}
+	s2orig := []complex128{1 + 1i, 2 + 2i, 3 + 3i}
+	dst1 := make([]complex128, 3)
+	ans := []complex128{2, 8, 18}
+	dst2 := MulConjTo(dst1, s1, s2)
+	if !EqualApprox(dst1, ans, EqTolerance) {
+		t.Errorf("MulConjTo doesn't give correct answer in mutated slice")
+	}
+	if !EqualApprox(dst2, ans, EqTolerance) {
+		t.Errorf("MulConjTo doesn't give correct answer in returned slice")
+	}
+	if !EqualApprox(s1, s1orig, EqTolerance) {
```

Comment:
This should test for exact equality.

Reply:
Done

There are others throughout the tests reflecting the original code in floats.